typing/typedecl.ml

```diff
@@ -1978,7 +1980,9 @@ let rec update_decl_jkind env dpath decl =
       decl
     | Type_open ->
       let type_jkind =
-        Jkind.Builtin.value ~why:Extensible_variant
+        Jkind.Builtin.non_float_value ~why:Extensible_variant
+        (* CR layouts: below is no longer true, and will be even less true
+           when exceptions start crossing portable/contended. Rethink? *)
```

Comment:
I think it's fine. The problem would be if we lower the kind without user intervention, but I don't think that's on the horizon. And getting this wrong isn't so bad -- it just means that it's more likely we have some work to do in our codebase when changing the language.